packages/modals/src/styled/StyledClose.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import classNames from 'classnames';
+import ModalStyles from '@zendeskgarden/css-modals';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'modals.close';
+
+export interface IStyledCloseProps {
+  hovered?: boolean;
```

Comment:
The `hovered` prop isn't renamed to use `isBoolean` naming convention because it will be removed in the CSS-in-JS migration.